lib/ansible/modules/cloud/docker/docker_compose.py

```diff
@@ -16,65 +16,67 @@
 
 module: docker_compose
 
-short_description: Manage docker services and containers.
+short_description: Manage docker containers with Docker Compose.
```

Comment:
How about:
```suggestion
short_description: Manage multi-container Docker applications with Docker Compose.
```
(Similarly to the first sentence here: https://docs.docker.com/compose/)